src/stockwatch/use_cases/configuring.py

```diff
@@ -17,12 +18,14 @@ class DeGiroServerConfig(ConfigSectionBase):
     login_url: str = "https://trader.degiro.nl/login/secure/login"
     client_url: str = "https://trader.degiro.nl/pa/secure/client"
     portfolio_url: str = (
-        "https://trader.degiro.nl/reporting/secure/v3/positionReport/csv"
+        "https://trader.degiro.nl/portfolio-reports/secure/v3/positionReport/csv"
     )
     account_url: str = (
-        "https://trader.degiro.nl/reporting/secure/v3/cashAccountReport/csv"
+        "https://trader.degiro.nl/portfolio-reports/secure/v3/cashAccountReport/csv"
+    )
+    user_agent: str = (
+        "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:107.0) Gecko/20100101 Firefox/107.0"
```

Comment:
Why did the formatting change here? Did we start using a different configuration?

Reply:
Same with all the added newlines after the docstrings

Reply:
Black version upgraded 23 -> 24 (these reflect the year); formatting rules change a bit during these upgrades. Either we stick with what we started with, or we accept a commit once a year that accomodates the new rules. I prefer the latter, so that formatting stays consistent over projects that I work on. Can you agree?